### DISCUSSION TITLE
Matrix Visuzaliazion Plugin

### DISCUSSION THREAD

**Discussion by raindancer2204** [ph1p/ikea-led-obegraensad#118] [Ideas]
Hi is it possible programm a like in the movie Matrix a visualization?

**Comment by jaal2001** [comment]
The rain animation could be a starting point for you. It does look bit like the matrix code, it is just not that big as the Matrix animation.

**Comment by raindancer2204** [comment]
Hi,
Unfortunately, since I'm not a big developer, this is something bigger for me. 

You should bring in good suggestions here, and give here a Info.

I can help with other actions such as the Readme update. 

I also wrote an update for VS Code installation. Like I said, I didn't want to upset anyone.
https://github.com/ph1p/ikea-led-obegraensad/issues/116

Someone would just have to add this to the readme and upload it.

Suggestions should be made and I did. 

I think it's a great project.
